jina/resources/k8s/template/deployment-gateway.yml

```diff
@@ -30,8 +30,7 @@ spec:
           imagePullPolicy: {pull_policy}
           command: {command}
           args: {args}
-          ports:
-            - containerPort: {port}
+          ports: {ports-section}
```

Comment:
why this change?

Reply:
we need a ports section with multiple ports here, rather than 1 port element.
This section is filled using a for loop to expose multiple ports
![image](https://user-images.githubusercontent.com/15269265/201936411-d5f84063-7b17-41e7-9e7f-0ed8bd0ae260.png)